### DISCUSSION TITLE
Cara mengisi missing value

### DISCUSSION THREAD

**Discussion by reniunisa** [teamalgoritma/community#446] [Q&A] [answered]
Halo,

Saya mencoba mengerjakan LBB classification dengan data airline survey. 
Salah 1 data int-nya, yaitu delay_arrival memiliki missing value 393 data dari data total 129.000 rows
Awalnya missing value saya abaikan, namun saat eksekusi ConfusionModel KNN, dia ga mau.

Error message: "no missing values are allowed"

Lalu saya coba isi pakai mean dari delay_arrival, tapi check mean hasilnya [NA]

Mohon petunjuknya

Terima kasih

**Comment by divakartika** [comment, marked as answer]
Dear @reniunisa 

Dapat ditampilkan variabel / kolom `delay_arrival` ini berisi data apa? Atau dapat dilampirkan link ke datasetnya?

**Reply by reniunisa** [reply]
saya gak pakai data dari file algoritma
pake data dari kaggle.com

**Reply by divakartika** [reply]
Baik bisa dilampirkan link ke datasetnya, Bu?

**Reply by reniunisa** [reply]
[Airline_survey.csv](https://github.com/teamalgoritma/community/files/9989871/Airline_survey.csv)
Terlampir ya mb :-)

**Reply by reniunisa** [reply]
Halo mb @divakartika, tadi udah dibantu di kelas mentoring dengan mb Fini dkk.
udah solved masalahnya.
Terima kasih banyak ya atas bantuan tim algoritma :-)

**Reply by divakartika** [reply]
Baik sama-sama, Bu :)
